src/shared/ui/Pagination/Pagination.tsx

```
@@ -29,12 +30,46 @@ export const Pagination = ({
   const visiblePages = getVisiblePages(currentPage, totalPages);
   const slots = style();
 
+  // 키보드 화살표로 페이지 이동
+    useEffect(() => {
```

ESLint

[error] Delete `··`

(prettier/prettier)